base/linalg/generic.jl

```diff
@@ -800,6 +800,8 @@ condition number, or componentwise relative condition number.
 """
 condskeel(A::AbstractMatrix, x::AbstractVector, p::Real=Inf) = norm(abs.(inv(A))*(abs.(A)*abs.(x)), p)
 
+issymmetric(A::AbstractMatrix{<:Real}) = ishermitian(A)
```

Comment:
I think this is the right way to define it since it is more likely that you'd define something Hermitian (or self adjoint). E.g. `AtA_or_AAt` and `SVDAugmented` are self adjoint by construction so we define `ishermitian(A::AtA_orAAt)=true`. If we remove the (correct) `issymmetric` fallback then it would be necessary to define `issymmetric` each of such operators.